flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/read/HiveTableInputFormat.java

```diff
@@ -170,8 +172,9 @@ private boolean useOrcVectorizedRead(HiveTablePartition partition) {
 			}
 		}
 
-		LOG.info("Use flink orc ColumnarRow reader.");
-		return true;
+		// temporarily disable the vectorized reader, until we figure out how to read timestamps with it
```

Comment:
No... I mean modify timestamp type, you can see above, there are support types.
I think you can use https://github.com/apache/flink/pull/10426 to verify and wait a minute to merge.